F/FLEXPART/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FLEXPART"
+version = v"10.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://www.flexpart.eu/gitmob/flexpart", "3d7eebf7c4909f59db5ec32c524f88fb846e9fe5"),
+    GitSource("https://www.flexpart.eu/gitmob/flex_extract", "e0005c99ac81d12faa45a8ff799debbd592b0dc0"),
```

Comment:
I just noticed there are two sources here: why these aren't two different packages?

Reply:
You're right, I will split both and create another branch for flex_extract, thanks for your comment!